kontrakter/kriminalomsorg/returPersonundersoekelse/1.0/eksempelfiler/returPersonundersoekelse-eksempel-1.json

```diff
@@ -0,0 +1,43 @@
+{
+  "forsendelse": {
+    "meldingsId": "C5E8E429-4321-48A9-A9D3-FDAA5F9BE2",
+    "sendtTid": "2023-11-03T10:19:55+02:00",
+    "avsender": {
+      "organisasjon": {
+        "organisasjonsnummer": "911830868",
+        "navn": "Akershus friomsorgskontor"
+      },
+      "person": {
+        "fornavn": "Per",
+        "mellomnavn": "S.",
+        "etternavn": "Undersøkersen",
+        "kontakt": {
+          "telefonnummer": ["22 11 33 44"],
+          "epost": "epost@krimomsorg1.no"
+        }
+      }
+    },
+    "mottakerOrganisasjon": {
+      "organisasjonsnummer": "983997953",
+      "navn": "Innlandet politidistrikt"
+    }
+  },
+  "hovedStraffesaksnummer": "15434811",
+  "rekvisisjonPersonundersoekelseId": "0322B3E2-B12B-40B6-8940-799C48706123",
+  "returPersonundersoekelseId": "0322B3E2-B12B-40B6-1234-799C48706123",
+  "dokumenter": [
+    {
+      "internId": "123234567871",
+      "overskrift": "Personundersøkelse(PDF)",
+      "kategori": {
+        "kode": "Personundersoekelse"
```

Comment:
Vi har et kodeverk på dokumentkategori her https://github.com/domstolene/ESAS/blob/main/kodeverk/felles/dokumentkategori.json 

Kunne du lagt inn et innslag eks `PERSONUNDERSOEKELSE` der og referert til det i eksempelet her?

Kodeverket er ikke tatt i bruk pdd., men ligger på planen å få tatt det i bruk. Kjekt om vi legger til rette for det her så tidlig som mulig.